modules/data.land/R/match_pft.R

```diff
@@ -0,0 +1,53 @@
+##' Match model PFTs
+##' 
+##' Matches BETYdb species IDs to model-specific PFTs
+##' 
+##' @param bety_species_id  vector of BETYdb species IDs
+##' @param pfts             settings$pfts.  List of pfts with database matching based on name
+##' @param con              database connection
+##' 
+##' @return table of BETYdb PFT IDs matched to species IDs
+##' 
+match_pft <- function(bety_species_id, pfts, con){
```

Comment:
is this `pfts` going to be exactly the same with the settings? what about non-pft pfts like soil, should we handle them before or in the function? (e.g. soil case would fail for duplicated species check etc.)